community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CreateTest.scala

```diff
@@ -166,11 +160,11 @@ In this case we add a +Person+ label to the node as well.
 By providing Cypher an array of maps, it will create a node for each map.
 
 NOTE: When you do this, you can't create anything else in the same +CREATE+ clause.
-""",
+             """,
```

Comment:
The indentation is really strange for many of the strings in these files. It looks really ugly in the source code when there is zero spaces between the margin and the start of the strings. Is this due to some format dependencies for the manual compilation?

I think that this change (and the one on line 149) is due to scala style formatting. Do they impair the results, or is it just a step away from our convention?